pages/create.tsx

```diff
@@ -156,15 +160,22 @@ const Create = () => {
               )}
               <div style={{ marginBottom: "40px" }} />
 
-              <StyledLabel>제목</StyledLabel>
+              <StyledLabel>
+                제목
+                <OverLine>{title.length}/50</OverLine>
+              </StyledLabel>
               <StyledInput
                 ref={titleRef}
                 placeholder="제목을 입력하세요."
                 value={title}
-                onChange={(e) => setTitle(e.target.value)}
+                onChange={(e) => setTitle(e.target.value.substring(0, 50))}
               />
               {submit && title === "" ? (
                 <StyledErrorText>* 제목은 필수 입력값입니다.</StyledErrorText>
+              ) : title.length >= 50 ? (
```

Comment:
<img width="417" alt="image" src="https://user-images.githubusercontent.com/96400112/184800846-e6f5be7c-8ae5-4463-877b-c510b235fe29.png">

```suggestion
              ) : title.length > 50 ? (
```

50자 이내로 입력 가능해서 등호는 없어도 될 것 같습니당

Reply:
그럼 알림이 안떠요

Reply:
조이가 `>199` 로 해놔서 저도 `>49`로 하겠습니다.